### PR TITLE
feat: Added examples

### PR DESCRIPTION
**Pull request by rg911**
feat: Added examples

### CODE REVIEW

File: examples/symbol/aggregate.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+// Assuming Alice is the aggregate transaction's initiator. A fund lock transaction is required prior announcing the bonded transaction.
+const lockTransactionBuilder = new HashLockTransactionBodyBuilder({
+    mosaic: new UnresolvedMosaicBuilder({
+        mosaicId: new UnresolvedMosaicIdDto(mosaicId),
+        amount: new AmountDto(BigInt(2000000)),
+    }),
+    duration: new BlockDurationDto(BigInt(100)),
+    hash: new Hash256Dto(bondedTransaction.transactionHash),
+});
+const lock = factory.create(SymbolDeadline.createFromAdjustedValue(100), BigInt(100), lockTransactionBuilder);
```
Comment (fboucquez) [inline examples/symbol/aggregate.ts:69]: Is it possible to create relative deadlines (now+2 hours - epoch adjustment)? Static deadlines are not used when the network is runnin

File: examples/symbol/multisig.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const cosignatoryKeyPairs = [
+    new SymbolKeyPair(Key.createFromHex('C73DF84178896A5D63F91467E606A44B06AF3892DB4F55FAF9524DA902E0FF7E')),
+    new SymbolKeyPair(Key.createFromHex('FC998941EDC8979252523885E4524598441577A9CEBCF5F766D194A11A1C317E')),
+    new SymbolKeyPair(Key.createFromHex('3AD4475F288A90654033EC4A4DE86F44ED3E898A384BE3488A2F02B95CC9C772')),
+];
+
+/**
+ * Build Symbol transaction using the factory and sign.
+ */
+const network = new SymbolNetwork('testnet', 0x98, '3B5E1FA6445653C971A50687E75E6D09FB30481055E3990C84B25E9222DC1155'); // Specify network
```
Comment (fboucquez) [inline examples/symbol/multisig.ts:15]: Should the user load the network from the static list? Should we have a SymbolNetwork.Testnet static value?

File: examples/symbol/aggregate.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        new UnresolvedMosaicBuilder({
+            mosaicId: new UnresolvedMosaicIdDto(mosaicId),
+            amount: new AmountDto(BigInt(2000000)),
+        }),
+    ],
+    message: Converter.concat(Uint8Array.of(0x00), Converter.utf8ToUint8('This is Bob')),
+});
+
+// Create deadline. 1616694977 is Symbol testnet epoch adjustment.
+const deadline = SymbolDeadline.create(1616694977, 2, ChronoUnit.HOURS);
```
Comment (fboucquez) [inline examples/symbol/aggregate.ts:52]: Maybe for the next pr, 1616694977 epoch should be part of the Network object, like the generation hash

Comment (rg911) [reply examples/symbol/aggregate.ts:52]: yeah, agree

**Review by fboucquez — APPROVED** [review]
Conditional to have epoch adjustment in networks, also maybe a SymbolNetwork.TESTNET constant  (Other PRs)

**Pull request merged**